lib/OpenQA/Worker.pm

```diff
@@ -181,15 +181,13 @@ sub capabilities ($self) {
     }
 
     # determine memory limit
-    open(my $MEMINFO, '<', '/proc/meminfo');
-    for my $line (<$MEMINFO>) {
-        chomp $line;
+    my @meminfo = split /\n/, path('/proc/meminfo')->slurp;
```

Comment:
Should this be `$meminfo = path('/proc/meminfo')->open('<');`

Reply:
No. We don't need to keep any file handle open. We just read the complete file in one go.